### DISCUSSION TITLE
Adding two M300 commands in LOAD / UNLOAD macro.

### DISCUSSION THREAD

**Discussion by xoga74** [jschuh/klipper-macros#104] [Ideas]
Since i am changing filament regulary starting from à cold extruder, the temperature must first raise before the filament can be unloaded. 

By adding two M300 lines in the filament.cfg file in the _load_unload macro, the machine lets you know your filament is unloaded/loaded.  This way you can do something else while waiting for the extruder to achieve the right temperature....

**Comment by jschuh** [comment]
Yeah, It's a small change to the `_load_unload` macro plus the `M600`, `M701`, and `M702` that call it. I'll probably do it.

**Comment by xoga74** [comment]
Tx

 

Van: Justin Schuh ***@***.***> 
Verzonden: dinsdag 16 mei 2023 16:15
Aan: jschuh/klipper-macros ***@***.***>
CC: xoga74 ***@***.***>; Author ***@***.***>
Onderwerp: Re: [jschuh/klipper-macros] Adding two M300 commands in LOAD / UNLOAD macro. (Discussion #104)

 

Yeah, It's a small change to the _load_unload macro plus the M600, M701, and M702 that call it. I'll probably do it.

**Comment by xoga74** [comment]
Tx and also a big thank you for the whole work you already made…

 

Van: Justin Schuh ***@***.***> 
Verzonden: dinsdag 16 mei 2023 16:15
Aan: jschuh/klipper-macros ***@***.***>
CC: xoga74 ***@***.***>; Author ***@***.***>
Onderwerp: Re: [jschuh/klipper-macros] Adding two M300 commands in LOAD / UNLOAD macro. (Discussion #104)

 

Yeah, It's a small change to the _load_unload macro plus the M600, M701, and M702 that call it. I'll probably do it.

**Comment by jschuh** [comment]
Okay, this landed on main in [v0.8.0](https://github.com/jschuh/klipper-macros/releases/tag/v0.8.0).
